src/components/Main.scss

```diff
@@ -0,0 +1,8 @@
+.main {
```

Comment:
I think we should do this in another way. without having to depend on an untargetable div :/

Reply:
I talked about this with seba, we thought of some possible solutions and we thought this one was the simplest. This way I don't have to set width 80% to each component, and everyone shared that style. But idk.